deliveries/bluesky_milestone_1.md

```diff
@@ -0,0 +1,44 @@
+# Milestone Delivery :mailbox:
+
+> ⚡ Only the GitHub account that submitted the application is allowed to submit milestones. 
+> 
+> Don't remove any of the mandatory parts presented in bold letters or as headlines! Lines starting with `>`, such as this one, can be removed.
+
+**The invoice has been send out correctly for this milestone and the delivery is according to the official [milestone delivery guidelines](https://github.com/smart-contract-bounty/Support-Docs/blob/master/milestone-deliverables-guidelines.md).**  
+
+* **Application Document:**, please provide a link to the merged contract from the running buonty (the `.md` file in the [applications](https://github.com/smart-contract-bounty/Wasm-Bounty-01/tree/master/applications) directory).
```

Comment:
Could you provide the link to the application document here? It would be easier to review, also for the future.

Reply:
Thanks for the feedback!  Addressing all in some steps:

1. Grammar errors - ran both chatGPT and ProWritingAid, updated the Medium article, and [checked in the pdf file](https://github.com/InkSmartContract/BlockchainFoodOrder/commit/cc2c34fcca505d8ac0bda098e9505f61253a3b42) to repo

2. We wrote the long article in order to cover all aspects of this example. The first part also covers the use case and background. Going forward, in part2 and part3, we'll make it much shorter for readability.

Reply:
3. Regarding the list of ink! macros, there might some misunderstandings. Unlike system level macros that can be used across projects, this project is meant to define application level macros to capture business logic. 

Declarative macro
The transfer_from_contract_to_account macro is called to pay the restaurant and the courier respectively. 

Procedure macros
The create_food, update_food and delete_food  macros are defined as procedure macros. They are called in [restaurant_service.rs ](https://github.com/InkSmartContract/BlockchainFoodOrder/blob/main/contracts/foodorder/logic/impls/restaurant_service.rs). This is to show that in the same application, users can call these macros to simply the code via encapsulated logic. 

The point is to learn to use the macro from the example, even though the example itself may not call the same macro many times.  Hope this clarifies.

Reply:
> Could you provide the link to the application document here? It would be easier to review, also for the future.

Good catch. Added the [project link ](https://github.com/dan0ero/Grant-Milestone-Delivery/commit/9aba1e2636da915a43e232d261b762687a15fb15)